tests/test_kiwi.py

```diff
@@ -77,8 +78,12 @@ def test_kiwi_create_image(
 
     assert container_per_test.connection.file("kiwi/build-tests").exists
 
-    kiwi_cmd = "kiwi-ng system build --description kiwi/build-tests/x86/leap/test-image-disk --set-repo obs://openSUSE:Leap:15.5/standard --target-dir /tmp/myimage"
-    container_per_test.connection.check_output(kiwi_cmd)
+    kiwi_cmd = "kiwi-ng system build --description kiwi/build-tests/x86/leap/test-image-disk --set-repo obs://openSUSE:Leap:15.5/standard --target-dir /var/tmp/myimage"
```

Comment:
why the change from `/tmp` to `/var/tmp` ?

Reply:
stupid oversight on my end, it's a leftover from some experimentation